manifests/charts/istio-cni/templates/daemonset.yaml

```diff
@@ -141,6 +145,10 @@ spec:
           imagePullPolicy: {{ .Values.cni.pullPolicy | default .Values.global.imagePullPolicy }}
 {{- end }}
           command: ["/opt/local/bin/istio-cni-taint"]
+          securityContext:
+            runAsUser: 1337
+            runAsGroup: 1337
+            runAsNonRoot: true
```

Comment:
Might make sense to add readOnlyRootFilesystem: true here. But otherwise, changes look good. Thanks @bianpengyuan.